.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -4,4 +4,4 @@ about: Suggest an idea for this project
 
 ---
 
-Feature requests are not handled in the repository. The best place to discuss these ideas would be on the /tg/station 13 forums here: https://tgstation13.org/phpBB/viewforum.php?f=9&sid=5153c1c704a4fb1006bf7a265e45e03f
+Feature requests are not handled in the repository. The best place to discuss these ideas would be on the beestation forums here: https://forums.beestation13.com/
```

Comment:
Except they are. They are and should be requested in a form of a github issue.

Reply:
If crossed wants to completly remove this I'll push the change, but it's like that for a really long time now.

Reply:
Afaik we're fine with feature requests on GitHub. Any opinions, @ike709 ?

Reply:
github is fine